src/columns.rs

```diff
@@ -77,14 +77,14 @@ where
 
     let inner_width = options
         .width
-        .saturating_sub(display_width(left_gap))
-        .saturating_sub(display_width(right_gap))
-        .saturating_sub(display_width(middle_gap) * (columns - 1));
+        .saturating_sub(display_width(left_gap, options.tab_width))
+        .saturating_sub(display_width(right_gap, options.tab_width))
+        .saturating_sub(display_width(middle_gap, options.tab_width) * (columns - 1));
 
     let column_width = std::cmp::max(inner_width / columns, 1);
     options.width = column_width;
     let last_column_padding = " ".repeat(inner_width % column_width);
-    let wrapped_lines = wrap(text, options);
+    let wrapped_lines = wrap(text, options.clone());
```

Comment:
I suppose you could save the `clone()` call if you copy out `options.tab_width` earlier in the function?

Reply:
Also fixed by 08e4d4a15c47a9093ef8e2afcb0bf2b0528941a1.